app/src/test/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfileTest.kt

```diff
@@ -0,0 +1,36 @@
+package org.jellyfin.androidtv.util.profile
+
+import org.jellyfin.androidtv.constant.CodecTypes
+import org.junit.Test
+import org.junit.Assert.assertTrue
+
+class ExoPlayerProfileTest {
```

Comment:
I don't think testing hardcoded values which can change frequently is useful here.